register.php

```diff
@@ -48,6 +52,8 @@
 				<label for = 'login'>Password*</label><br />
 				<!-- input box for the user to enter their password -->
 				<input type = "password" name = "password" id="myInput" placeholder="Enter Password"/><br />
+				<!-- An element to toggle between password visibility -->
+				<input type="checkbox" onclick="myFunction()">Show Password<br /><br />
```

Comment:
#show password button so the user can see what they have entered as their password